igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```
@@ -271,24 +262,27 @@ class BuildController {
         }
     }
 
-    private String getArtifactPathFromBuild(jenkinsService, job, buildNumber, String fileName) {
-        try {
-            return retrySupport.retry({ ->
-                def artifact = jenkinsService.getBuild(job, buildNumber).artifacts.find {
-                    it.fileName == fileName
-                }
-                if (artifact) {
-                    return artifact.relativePath
-                }
-                log.info("Could not find artifact {} for job {} on build #{}", fileName, job, buildNumber)
-                throw new ArtifactNotFoundException()
-            }, 5, 2000, false)
-        } catch (ArtifactNotFoundException ignored) {
-            return null
```

I removed this empty catch block so the exception propagates out of here